ui/shared/components/panel/variants/SavedVariants.jsx

```diff
@@ -74,17 +72,15 @@ class SavedVariants extends React.PureComponent {
 
   render() {
     const {
-      match, tableState, filters, discoveryFilters, totalPages, variantsToDisplay, totalVariantsCount, firstRecordIndex,
+      match, tableState, filters, totalPages, variantsToDisplay, totalVariantsCount, firstRecordIndex,
       tableSummaryComponent, loading, filteredVariantsCount, tagOptions, additionalFilter, updateTableField,
-      variantExportConfig, loadVariants, error,
+      variantExportConfig, loadVariants, error, multiple, getSelectedTag,
     } = this.props
     const { showAllFilters } = this.state
-    const { familyGuid, variantGuid, tag } = match.params
-
-    const appliedTagCategoryFilter = tag || (variantGuid ? null : (tableState.categoryFilter || ALL_FILTER))
+    const { familyGuid, variantGuid } = match.params
+    const optionValue = getSelectedTag(variantGuid ? null : (tableState.categoryFilter || ALL_FILTER))
```

Comment:
Use a function prop to get the selected option value because the `tableState` is only available here.